packages/principal/src/account.js

```diff
@@ -0,0 +1,41 @@
+import * as Signature from '@ipld/dag-ucan/signature'
+import * as UCAN from '@ipld/dag-ucan'
+
+/**
+ * @template {UCAN.DID} ID
+ * @param {{id: ID }} id
+ * @returns {UCAN.Signer<ID, Signature.NON_STANDARD>}
+ */
+export const from = ({ id }) => new Account(id)
+
+/**
+ * An account is a special type of signer that produces empty signature, which
+ * signals that verifier needs to verify authorization interactively.
+ *
+ * @template {UCAN.DID} ID
+ * @implements {UCAN.Signer<ID, Signature.NON_STANDARD>}
+ */
+class Account {
```

Comment:
nit: if possible I'd avoid using `Account` at all here (it's too easy to misinterpret) and call this something like `NonStandardSigner`. just a thought

Reply:
Renamed it to `Absentee` which I think reflect the better than either misleading `Account` or `NonStandardSigner` which isn't specific enough. I'll assume it's ok, if not we can rename in the followup change.